### DISCUSSION TITLE
[mini.indentscope] Only trigger indentscope inside specific filetypes (+color change)

### DISCUSSION THREAD

**Discussion by lsig** [nvim-mini/mini.nvim#311] [Q&A] [answered]
I was wondering if it was possible to specify the filetypes the indentscope gets activated and also if it is possible to change the color of the line from green to grey?

**Comment by echasnovski** [comment, marked as answer]
> I was wondering if it was possible to specify the filetypes the indentscope gets activated and also if it is possible to change the color of the line from green to grey?

"Yes" to both questions.

Any 'mini.nvim' module can be disabled per buffer by setting buffer-local variable in the Lua form of `vim.b.minixxx_disable = true` (`miniindentscope_disable` in this case). Here is a [list of common recipes](https://github.com/echasnovski/mini.nvim/blob/main/doc/mini.txt#L269) for its automatic usage.

Color of that line is controlled by `MiniIndentscopeSymbol` highlight group (described in ['mini.indentscope' documentation](https://github.com/echasnovski/mini.nvim/blob/main/doc/mini-indentscope.txt#L55)). You can modify it to be any color using regular `:highlight` command or its `vim.api.nvim_set_hl()` Lua equivalent.
